Snyk PR demo

wmurphysnyk opened the pull request.

snyk-bot committed:
fix: package.json & package-lock.json to reduce vulnerabilities

The following vulnerabilities are fixed with an upgrade:
- https://snyk.io/vuln/SNYK-JS-VM2-3018201

wmurphysnyk committed:
Merge pull request #62 from wmporrassnyk/snyk-fix-a34ad352fde07302d00516e825c88ef6

[Snyk] Security upgrade juicy-chat-bot from 0.6.5 to 0.6.6

snyk-bot committed:
fix: package.json & package-lock.json to reduce vulnerabilities

The following vulnerabilities are fixed with an upgrade:
- https://snyk.io/vuln/SNYK-JS-VM2-5415299

wmurphysnyk committed:
Merge pull request #73 from wmporrassnyk/snyk-fix-908f9cc197e35e1d490f4c02096e295e

[Snyk] Security upgrade juicy-chat-bot from 0.6.6 to 0.7.1

snyk-bot committed:
fix: package.json & package-lock.json to reduce vulnerabilities

The following vulnerabilities are fixed with an upgrade:
- https://snyk.io/vuln/npm:sanitize-html:20141024

wmurphysnyk committed:
Merge pull request #75 from wmporrassnyk/snyk-fix-763169909c8d77eb518626fa524bc79a

[Snyk] Security upgrade sanitize-html from 1.4.2 to 1.4.3

snyk-bot committed:
fix: package.json & package-lock.json to reduce vulnerabilities

The following vulnerabilities are fixed with an upgrade:
- https://snyk.io/vuln/SNYK-JS-SEQUELIZE-2932027

wmurphysnyk committed:
Merge pull request #76 from wmporrassnyk/snyk-fix-a1f037c43fe17d2019b0236b2cbc7837

[Snyk] Security upgrade sequelize from 5.22.5 to 6.19.1

snyk-bot committed:
fix: package.json & package-lock.json to reduce vulnerabilities

The following vulnerabilities are fixed with an upgrade:
- https://snyk.io/vuln/npm:moment:20170905

wmurphysnyk committed:
Merge pull request #77 from wmporrassnyk/snyk-fix-0d355dd5ff1007f779dd85e5b856de78

[Snyk] Security upgrade express-jwt from 0.1.3 to 0.1.4

snyk-bot committed:
fix: package.json & package-lock.json to reduce vulnerabilities

The following vulnerabilities are fixed with an upgrade:
- https://snyk.io/vuln/SNYK-JS-SANITIZEHTML-585892

wmurphysnyk committed:
Merge pull request #87 from wmporrassnyk/snyk-fix-7e94e3e5f84d686a34abb6eb49511038

[Snyk] Security upgrade sanitize-html from 1.4.3 to 2.0.0

snyk-bot committed:
fix: package.json & package-lock.json to reduce vulnerabilities

The following vulnerabilities are fixed with an upgrade:
- https://snyk.io/vuln/npm:base64url:20180511

wmurphysnyk committed:
Merge pull request #88 from wmporrassnyk/snyk-fix-015701b44d62efc7185ed530e9187c6e

[Snyk] Fix for 1 vulnerabilities

snyk-bot committed:
fix: frontend/package.json & frontend/package-lock.json to reduce vulnerabilities

The following vulnerabilities are fixed with an upgrade:
- https://snyk.io/vuln/SNYK-JS-ANSIHTML-1296849

wmurphysnyk committed:
Merge pull request #90 from wmporrassnyk/snyk-fix-ceedff3b40ea185199902ff4d0be4036

[Snyk] Security upgrade @angular-devkit/build-angular from 0.1000.8 to 0.1102.18

snyk-bot committed:
fix: package.json & package-lock.json to reduce vulnerabilities

The following vulnerabilities are fixed with an upgrade:
- https://snyk.io/vuln/SNYK-JS-SQLITE3-3358947

wmurphysnyk committed:
Merge pull request #92 from wmporrassnyk/snyk-fix-f74a965e305fce8f89ca9b73d648bee4

[Snyk] Security upgrade sqlite3 from 5.0.2 to 5.1.5

snyk-bot committed:
fix: Dockerfile to reduce vulnerabilities

The following vulnerabilities are fixed with an upgrade:
- https://snyk.io/vuln/SNYK-DEBIAN10-LIBWEBP-5893093
- https://snyk.io/vuln/SNYK-DEBIAN10-LIBWEBP-5893093
- https://snyk.io/vuln/SNYK-DEBIAN10-LIBWEBP-5893093
- https://snyk.io/vuln/SNYK-DEBIAN10-LIBWEBP-5893093
- https://snyk.io/vuln/SNYK-DEBIAN10-NGHTTP2-5953390

wmurphysnyk committed:
Merge pull request #95 from wmporrassnyk/snyk-fix-15e2602d9f059d5f143f2c84e21fb8f5

[Snyk] Security upgrade node from 12.18.4-buster to 21.5.0-slim

snyk-bot committed:
fix: package.json & package-lock.json to reduce vulnerabilities

The following vulnerabilities are fixed with an upgrade:
- https://snyk.io/vuln/SNYK-JS-SANITIZEHTML-6256334

wmurphysnyk committed:
Create main.yml

wmurphysnyk committed:
Merge pull request #99 from wmporrassnyk/snyk-fix-e933255e03ded083283e679a156b275d

[Snyk] Security upgrade sanitize-html from 2.0.0 to 2.12.1

snyk-bot committed:
fix: package.json & package-lock.json to reduce vulnerabilities

The following vulnerabilities are fixed with an upgrade:
- https://snyk.io/vuln/SNYK-JS-GRUNT-2813632

wmurphysnyk committed:
Merge pull request #100 from wmporrassnyk/snyk-fix-49704ab5c2e276b74de596a9b66fa341

[Snyk] Security upgrade grunt from 1.4.1 to 1.5.3

snyk-bot committed:
fix: frontend/package.json to reduce vulnerabilities

The following vulnerabilities are fixed with an upgrade:
- https://snyk.io/vuln/SNYK-JS-TAR-6476909
- https://snyk.io/vuln/SNYK-JS-WEBPACKDEVMIDDLEWARE-6476555

wmurphysnyk committed:
Merge pull request #104 from wmporrassnyk/snyk-fix-5353491209559f7a7274c495bab36b3a

[Snyk] Fix for 2 vulnerabilities

snyk-bot committed:
fix: package.json & package-lock.json to reduce vulnerabilities

The following vulnerabilities are fixed with an upgrade:
- https://snyk.io/vuln/SNYK-JS-SEQUELIZE-2959225

wmurphysnyk committed:
Merge pull request #106 from wmporrassnyk/snyk-fix-724dc4105140c1895dcf10e9cc699e62

[Snyk] Security upgrade sequelize from 6.19.1 to 6.21.2

snyk-bot committed:
fix: package.json & package-lock.json to reduce vulnerabilities

The following vulnerabilities are fixed with an upgrade:
- https://snyk.io/vuln/SNYK-JS-SEQUELIZE-3324088

wmurphysnyk committed:
Merge pull request #107 from wmporrassnyk/snyk-fix-3c400f3d9a0d1f9266dc042e6e758bf0

[Snyk] Security upgrade sequelize from 6.21.2 to 6.29.0

snyk-bot committed:
fix: package.json & package-lock.json to reduce vulnerabilities

The following vulnerabilities are fixed with an upgrade:
- https://snyk.io/vuln/SNYK-JS-ENGINEIO-3136336

wmurphysnyk committed:
Merge pull request #110 from wmporrassnyk/snyk-fix-ee1205932d29e49e89d5646cf4536aee

[Snyk] Security upgrade socket.io from 2.4.1 to 2.5.0

snyk-bot committed:
fix: upgrade semver from 7.3.5 to 7.6.0

Snyk has created this PR to upgrade semver from 7.3.5 to 7.6.0.

See this package in npm:
https://www.npmjs.com/package/semver

See this project in Snyk:
https://app.snyk.io/org/wmporrassnyk/project/e161c865-a262-4b6d-a030-b0a157cf2a4a?utm_source=github&utm_medium=referral&page=upgrade-pr

wmurphysnyk committed:
Merge pull request #115 from wmporrassnyk/snyk-upgrade-566e6ac3e65bb47b396c98baa158124f

[Snyk] Upgrade semver from 7.3.5 to 7.6.0

snyk-bot committed:
fix: package.json & package-lock.json to reduce vulnerabilities

The following vulnerabilities are fixed with an upgrade:
- https://snyk.io/vuln/SNYK-JS-CRYPTOJS-6028119

wmurphysnyk committed:
Merge pull request #119 from wmporrassnyk/snyk-fix-ee8b01f14fb25736c02e1871df531917

[Snyk] Security upgrade pdfkit from 0.11.0 to 0.12.2

wmurphysnyk committed:
Create npm-grunt.yml

ADD IN BUILD SNYK STEPS

wmurphysnyk committed:
Create Snyk build

wmurphysnyk committed:
Update npm-grunt.yml

wmurphysnyk committed:
Delete .github/workflows/Snyk build

wmurphysnyk committed:
Delete .github/workflows/main.yml

wmurphysnyk committed:
Create main.yml

wmurphysnyk committed:
Update npm-grunt.yml

wmurphysnyk committed:
Delete .github/workflows/npm-grunt.yml

wmurphysnyk committed:
Update main.yml

wmurphysnyk committed:
Create .snyk

snyk-bot committed:
fix: upgrade @nguniversal/express-engine from 10.0.1 to 10.1.0

Snyk has created this PR to upgrade @nguniversal/express-engine from 10.0.1 to 10.1.0.

See this package in npm:
@nguniversal/express-engine

See this project in Snyk:
https://app.snyk.io/org/wmporrassnyk/project/113e2291-a448-4044-96fe-3988961a7f27?utm_source=github&utm_medium=referral&page=upgrade-pr

wmurphysnyk committed:
Merge pull request #372 from wmurphysnyk/snyk-upgrade-84560e4ad09c18ab3d586340dd00aa06

[Snyk] Upgrade @nguniversal/express-engine from 10.0.1 to 10.1.0

snyk-bot committed:
fix: upgrade socket.io-client from 2.4.0 to 2.5.0

Snyk has created this PR to upgrade socket.io-client from 2.4.0 to 2.5.0.

See this package in npm:
socket.io-client

See this project in Snyk:
https://app.snyk.io/org/wmporrassnyk/project/113e2291-a448-4044-96fe-3988961a7f27?utm_source=github&utm_medium=referral&page=upgrade-pr

snyk-bot committed:
fix: upgrade sequelize from 6.29.0 to 6.37.6

Snyk has created this PR to upgrade sequelize from 6.29.0 to 6.37.6.

See this package in npm:
sequelize

See this project in Snyk:
https://app.snyk.io/org/wmporrassnyk/project/e161c865-a262-4b6d-a030-b0a157cf2a4a?utm_source=github&utm_medium=referral&page=upgrade-pr

wmurphysnyk committed:
Merge pull request #379 from wmurphysnyk/snyk-upgrade-db3bdf9971b6aa7f95bc6be7ff0bf1a3

[Snyk] Upgrade sequelize from 6.29.0 to 6.37.6

wmurphysnyk committed:
Merge pull request #377 from wmurphysnyk/snyk-upgrade-79f82729f1fc94c8882488ad9536869b

[Snyk] Upgrade socket.io-client from 2.4.0 to 2.5.0

wmurphysnyk committed:
Update package.json

"lodash": "4.17.20"
added

alexeisnyk commented:
### :warning: **Snyk checks are incomplete.**

| Status | Scanner | ![](https://res.cloudinary.com/snyk/image/upload/r-d/scm-platform/snyk-pull-requests/severity-critical.svg) Critical | ![](https://res.cloudinary.com/snyk/image/upload/r-d/scm-platform/snyk-pull-requests/severity-high.svg) High | ![](https://res.cloudinary.com/snyk/image/upload/r-d/scm-platform/snyk-pull-requests/severity-medium.svg) Medium | ![](https://res.cloudinary.com/snyk/image/upload/r-d/scm-platform/snyk-pull-requests/severity-low.svg) Low | Total (2) |
|---|---|---|---|---|---|---|
| :warning: | Open Source Security| 0 | 0 | 2 | 0 | [ See details ](https://app.snyk.io/org/demo-org/pr-checks/2ad8efe9-3fea-43ae-a42e-936f04b65a43?refs=02b821a755acc2315a54e4669a3657c16e6a083e&source=prChecksComment) |
| :warning: | Licenses| 0 | 0 | 0 | 0 | [ See details ](https://app.snyk.io/org/demo-org/pr-checks/2ad8efe9-3fea-43ae-a42e-936f04b65a43/license?refs=02b821a755acc2315a54e4669a3657c16e6a083e&source=prChecksComment) |

:computer: Catch issues earlier using the plugins for [VS Code](https://marketplace.visualstudio.com/items?itemName=snyk-security.snyk-vulnerability-scanner), [JetBrains IDEs](https://plugins.jetbrains.com/plugin/10972-snyk-security), [Visual Studio](https://marketplace.visualstudio.com/items?itemName=snyk-security.snyk-vulnerability-scanner-vs-2022), and [Eclipse](https://marketplace.eclipse.org/content/snyk-security#external-install-button).

snyk-bot committed:
fix: upgrade core-js from 3.22.8 to 3.47.0

Snyk has created this PR to upgrade core-js from 3.22.8 to 3.47.0.

See this package in npm:
core-js

See this project in Snyk:
https://app.snyk.io/org/wmporrassnyk/project/113e2291-a448-4044-96fe-3988961a7f27?utm_source=github&utm_medium=referral&page=upgrade-pr

wmurphysnyk committed:
Fail PrCheck Demo adding lodash dependency to package.json

Fail PrCheck Demo adding lodash dependency to package.json

wmurphysnyk committed:
Merge pull request #404 from wmurphysnyk/snyk-upgrade-d0d7d5ab9463449b5f337e38ad1360aa

[Snyk] Upgrade core-js from 3.22.8 to 3.47.0
